tool/site/server/src/app.ts

```diff
@@ -45,25 +47,27 @@ function createServer() {
 }
 
 async function registerAuthentication(server: Hapi.Server) {
-  await server.register(Cookie)
-  server.auth.strategy("session", "cookie", {
-    cookie: {
-      name: "auth",
-      password: CONFIG.sessionKey,
-      isSecure: true
-    },
-    redirectTo: (request: Hapi.Request) => {
-      if (isDev(CONFIG.environment)) {
-        setSessionValue(
-          "use_signing_mock",
-          request.query["use_signing_mock"],
-          request
-        )
+  if (!isSandbox(CONFIG.environment)) {
```

Comment:
with the above change (lines 18-20) this function is only called when not in sandbox mode, so not sure we need this check here?

Reply:
Good spot, forgot I'd already made that change some time ago. Will remove duplication